### PR TITLE
Iae 16657

### PR DESCRIPTION
**Pull request by thomastighe**

### CODE REVIEW

File: apps/sam-design-system-site/src/app/app.routes.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Routes, RouterModule } from '@angular/router';
+
+export const ROUTES: Routes = [
+  { path: 'component', loadChildren: '@gsa-sam/component-sample#ComponentSampleModule' } 
+ //{path:'garbage', loadChildren:'@gsa-sam/fakenew#FakenewModule'}
```
Comment (colinalford-gsa) [inline apps/sam-design-system-site/src/app/app.routes.ts]: @thomastighe Please trash this!

File: apps/sam-design-system-site/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -14,18 +15,18 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'sam-design-system-site'`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.debugElement.componentInstance;
-    expect(app.title).toEqual('sam-design-system-site');
-  });
+  // it(`should have as title 'sam-design-system-site'`, () => {
```
Comment (colinalford-gsa) [inline apps/sam-design-system-site/src/app/app.component.spec.ts]: @thomastighe Please trash  these commented tests!

File: apps/sam-design-system-site/src/app/app.routes.ts (hunk 1 above)
Comment (thomastighe) [reply apps/sam-design-system-site/src/app/app.routes.ts]: Changed

File: apps/sam-design-system-site/src/app/app.component.spec.ts (hunk 2 above)
Comment (thomastighe) [reply apps/sam-design-system-site/src/app/app.component.spec.ts]: Changed

**Pull request merged**